templates/students/attendance.html

```diff
@@ -43,7 +43,8 @@ <h3 class="text-center">Students Attendance</h3>
                 <tbody>
                     {% for stu_att in stu_att_today %}
                     <tr>
-                        <td>{{ stu_att.student.get_full_name }}</td>
+                       <a href="{% url 'students:profile' stu_att.student.id     %}">
+                        <td>{{ stu_att.student.get_full_name }}</td></a> 
```

Comment:
On selecting a different class group from top dropdown, this would fail as the data is updated from the javascript in that case. So you need to fix this too.

You can take a look at #134, which resolves a similar problem.

Reply:
I have updated the changes and pushed the code, have a look @garg3133

Reply:
The links are not working, neither on the dashboard nor on the attendance page. Please test your code first thoroughly and then update the PR and ask for review.

Reply:
Can you guide me, I was not much aware of the front end workflow , I tried and followed the PR ref that you mentioned
Can you tell me what, where do I need to add

Reply:
![LIS](https://user-images.githubusercontent.com/56113657/120097870-71757680-c150-11eb-885c-e9a566c98c3b.png)
List of students are not visible, How can I test

Reply:
First go to the student attendance page (http://127.0.0.1/students/attendance/), you will see there that the names of the students are not highlighted,

If you take the attendance of a few students and then go to dashboard (http://127.0.0.1/dashboard), click on the submit button on the dashboard. At the bottom, you'll see a list of students whom you marked as present on the attendance page. Here also, name of the students is not clickable.